xml/System.Net/ServicePoint.xml

```diff
@@ -400,9 +400,9 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- If the <xref:System.Net.ServicePoint> object was constructed with a <xref:System.Uri> object (the <xref:System.Net.ServicePointManager.FindServicePoint%2A> or <xref:System.Net.ServicePointManager.FindServicePoint%2A> methods, for example), then the <xref:System.Net.ServicePoint.ConnectionName%2A> property represents the <xref:System.Uri.Scheme%2A> property of the <xref:System.Uri> object used.  
+ If the <xref:System.Net.ServicePoint> object was constructed with a <xref:System.Uri> object (the <xref:System.Net.ServicePointManager.FindServicePoint> or <xref:System.Net.ServicePointManager.FindServicePoint> methods, for example), then the <xref:System.Net.ServicePoint.ConnectionName> property represents the <xref:System.Uri.Scheme> property of the <xref:System.Uri> object used.  
```

Comment:
Both links to FindServicePoint are broken because the %2A has been removed. But these are also links to the same overloaded method. The original intent was probably to link to two overloads of FindServicePoint with URL arguments. In that case, the full parameter list of each method should be provided, and ?displayProperty=nameWithType should be added:
- \<xref:System.Net.ServicePointManager.FindServicePoint(System.Uri,System.Net.IWebProxy)?displayProperty=nameWithType>.
- \<xref:System.Net.ServicePointManager.FindServicePoint(System.Uri)?displayProperty=nameWithType>

The other option is to modify the beginning of the sentence so it reads something like, "If the \<xref:System.Net.ServicePoint> object was constructed by calling a \<xref:System.Net.ServicePointManager.FindServicePoint%2A> overload with a \<xref:System.Uri> argument, then...